governance-and-automation/automated-workflows/policy-automation.md

```diff
@@ -0,0 +1,84 @@
+
+{% hint style="important" %}
+**Please Note!**
+
+To activate rules for auto-applying policies, please make sure you have already set up your policies under the automated workflow section. [Follow this article for more information. ](../automated-workflows/set-up-automated-workflows.md)
+
+{% endhint %}
+
+
+## Create Rules
+
+To create rules for auto-applying policies, take the following steps:
+
+* Open your Syskit Point and **navigate to Settings** and under **Governance** select **Policy Automation (1)**
+* **Click the New Rule button (2)** and a pop-up will appear
+
+![Policy Automation - Create Rule](../../.gitbook/assets/policy-automation_policy-settings.png)
+
+To create the rule, fill out the following information, as requested on the pop-up:
+
+* Under the General section, fill out the following:
+  * **Rule Name (1)** - in this space, you can write your desired rule name
+  * **Description (2)** - this space can be used to describe the rule which you want to create
+* Under the **Priority section (3)**, set your desired level of priority for this rule by **selecting a number (4)** between 1-6 from the drop-down menu
+  * Priority will be **set as 1 by default** and is the number that signifies the highest priority, with 6 being the lowest priority
+  * This change also affects priorities on previously defined rules
+* When ready, **click Next to continue (5)**
+
+![Policy Automation - New Rule](../../.gitbook/assets/policy-automation_new-policy-first.png)
+
+
+* Under the **Add Conditions (1)** section, **set the conditions for your rule (2)** by using the drop-down menu
+  * Once you've set the condition, **click the Save Condition (3)** button
+  * Once saved, if you want to make changes **click the edit button** 
+  * Once saved, if you want to delete it, **click the delete button**
+* When you're ready, **click Next to continue (4)**
+
+![Policy Automation - New Rule - Add Condittions](../../.gitbook/assets/policy-automation_new-policy-second.png)
+
+* Under the **Apply Policies (1)** section, you can select which policies the rule should apply for 
+  * In the section **Apply following policies** select the **Add Policy (2)** button and **select your desired policy (3)** from the drop-down menu available
+  * Once you've selected the policy or policies, **click the Save Policy (4)** button
+  * If you've set a policy but want to change it,  **click the edit button** 
+  * If you've set a policy but want to delete it, **click the delete button**
+* **Click the preview button (5)** when your preferences have been set to see how this new rule would affect your workspaces
+   * By clicking preview, you can also see if there are **any “Priority Overides”** on any of your workspace
```

Comment:
Možda objasniti malo više taj preview npr koji su Priority Overrides i idu dva r u Overrides. Preview button isto velikim ako mu je to ime